frontend/src/components/Consent/Consent.js

```diff
@@ -15,7 +15,7 @@ const Consent = ({ title, text, experiment, participant, onNext, confirm, deny,
         if (consent || (new URLSearchParams(url_query_string).get("participant_id"))) {
             onNext();
         }
-    }, [consent, onNext]);
+    }, [consent, onNext, url_query_string]);
```

Comment:
Why is it needed to add `url_query_string` as an effect dependency? URL query string will not change during the experiment, and Consent will be re-rendered anyway at the beginning of the next experiment. It will not hurt, but seems unnecessary.
Otherwise, everything seems fine!

Reply:
This came from a React warning. Basically, any variable in a `useEffect` function is considered being subject to change.